server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/routes/HealthCheckRoutesTest.java

```diff
@@ -131,4 +139,100 @@ public void validateHealthchecksShouldReturnInternalErrorWhenOneHealthCheckIsDeg
         .then()
             .statusCode(HttpStatus.INTERNAL_SERVER_ERROR_500);
     }
+    
+    @Test
+    public void performHealthCheckShouldReturnOkWhenHealthCheckIsHealthy() {
+        healthChecks.add(healthCheck(Result.healthy(COMPONENT_NAME_1)));
+        
+        given()
+            .pathParam("componentName", COMPONENT_NAME_1.getName())
+        .when()
+            .get("/checks/{componentName}")
+        .then()
+            .statusCode(HttpStatus.OK_200)
+            .body("componentName", equalTo("component-1"))
```

Comment:
could you extract "component-1" into a constant? I see it is used in many places.

Reply:
fixed in d9e182e5973b650ab3effed6a6aeba97f884a665